autoload/indent_guides.vim

```diff
@@ -185,7 +185,7 @@ function! indent_guides#init_script_vars()
   let s:hi_normal   = indent_guides#capture_highlight('Normal')
 
   " remove 'font=<value>' from the s:hi_normal string (only seems to happen on Vim startup in Windows)
-  let s:hi_normal = substitute(s:hi_normal, ' font=[A-Za-z0-9:]\+', "", "")
+  let s:hi_normal = substitute(s:hi_normal, ' font=[A-Za-z0-9: ]\+', "", "")
```

Comment:
This would be too greedy because other segments are space separated.
A proper match (if such a thing is even still needed at all) wourd need to
account for whatever quoting or escaping is used in this value. Since
I don't have a working test case I can't say what that needs to be.